lib/search/query.rb

```diff
@@ -25,7 +25,36 @@ def run(search_params)
 
       payload     = process_elasticsearch_errors { builder.payload }
       es_response = process_elasticsearch_errors { index.raw_search(payload) }
+      process_es_response(search_params, builder, payload, es_response)
+    end
+
+  private
```

Comment:
I don't think very much has changed here.  I wonder if it's possible to split the method extraction into a separate commit.  It might make it easier to reason about when we come back to it in future?